File: design-pattern-java/app/src/main/java/design/pattern/java/patterns/structural/bridge/SomeLibrary.java
package design.pattern.java.patterns.structural.bridge;

public class SomeLibrary {
    public SomeLibrary() {}
    public String writeString() {
        return "String wrote";
    }
    public Integer integerGenerator() {
        return Double.valueOf(Math.random() * 100).intValue();
    }
    public String writeOtherString() {
        return "Other string wrote";
    }
}

File: design-pattern-jvm/app/src/main/kotlin/design/pattern/kotlin/patterns/structural/bridge/SomeLibrary.kt
package design.pattern.kotlin.patterns.structural.bridge

open class SomeLibrary {
    open fun writeString(): String {
        return "String wrote"
    }

    open fun integerGenerator(): Int {
        return (Math.random() * 100).toInt()
    }

    open fun writeOtherString(): String {
        return "Other string wrote"
    }
}